### DISCUSSION TITLE
Incorrect validation of PRs

### DISCUSSION THREAD

**Discussion by CGQAQ** [microsoft/winget-pkgs#1453] [General]
Both #1101 and  #1102 have Validation-SmartScreen error, #1103 has validation error.
These three are all from Mozilla official site, I think it's misjudged

**Comment by latere-a-latere** [comment]
SmartScreen pops up when the applications don't have any reputation yet. Just submit the installers to Microsoft and post the result once cleared: https://www.microsoft.com/en-us/wdsi/filesubmission

**Comment by CGQAQ** [comment]
> SmartScreen pops up when the applications don't have any reputation yet. Just submit the installers to Microsoft and post the result once cleared: https://www.microsoft.com/en-us/wdsi/filesubmission

@MarNicGit why not just auto upload via the robot, you already have download link inside the yaml file

**Comment by latere-a-latere** [comment]
@CGQAQ You'll have to propose that to the maintainers

**Comment by CGQAQ** [comment]
@MarNicGit Maybe a new issue for that?

**Comment by denelon** [comment]
@CGQAQ we're already working on a better solution with some teams internal to Microsoft. If we get some positive results, the process should just get better :)

**Comment by CGQAQ** [comment]
> 
> 
> @CGQAQ we're already working on a better solution with some teams internal to Microsoft. If we get some positive results, the process should just get better :)

Oh! thanks :)
